src/script/components/android-form.ts

```diff
@@ -223,9 +233,9 @@ export class AndroidForm extends AppPackageFormBase {
               <div id="all-settings-header" slot="heading">
                 <span>All Settings</span>
 
-                <fast-button class="flipper-button" mode="stealth">
+                <div class="flipper-button" mode="stealth">
```

Comment:
Does this `mode` still work here? Also, for accessibility, should this be a button or atleast have an aria-role for that div?

Reply:
I removed the mode. I also added a role="button". Making it a <button> leads to the same issue of a button tag nested into a button. 

![Web capture_14-3-2022_111553_ghbhpcookfemncgoinjblecnilppimih](https://user-images.githubusercontent.com/34525412/158219797-2bd0df18-513a-4603-9d69-4da320116229.jpeg)